src/app/components/BibPage/BibPage.jsx

```diff
@@ -61,22 +60,21 @@ const BibPage = (props) => {
               aria-relevant="additions removals"
               aria-describedby="results-description"
             >
-              <BibMainInfo bib={bib} />
-            </div>
-
-            <div className="nypl-column-three-quarters">
               <div className="nypl-item-details">
-                <ItemHoldings
-                  shortenItems={shortenItems}
-                  items={items}
-                  bibId={bibId}
-                  title="AVAILABILITY"
-                />
-                <MarcRecord bNumber={bNumber[0]} />
-                <BibDetails
-                  bib={bib}
-                  title="Bib details"
-                />
+                <h1>{title}</h1>
+                <dl>
```

Comment:
`MarcRecord`, `BibDetails`, and `ItemHolidings` all return a `<dl>` wrapper so you don't need to include it here. If you do want to add it here instead, you'd have to update the rest of the components so make sure the DOM structure is correct when doing so.